core/src/test/java/org/jsmart/zerocode/core/domain/StepTest.java

```diff
@@ -161,7 +168,126 @@ public void shouldSerializeSingleStep_method() throws Exception {
         JsonNode singleStepNode = mapper.valueToTree(stepJava);
         String singleStepNodeString = mapper.writeValueAsString(singleStepNode);
 
-        JSONAssert.assertEquals(jsonDocumentAsString,singleStepNodeString, false);
+        JSONAssert.assertEquals(jsonDocumentAsString, singleStepNodeString, false);
         assertThat(singleStepNode.get("method").asText(), is("POST"));
     }
+
+    @Test
+    public void testPayLoad_strictPass() {
```

Comment:
Add a unit test for STRICT array matching for example
if expected and actual array has only one element with same values, then only the matching will pass

Reply:
done